recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "tox" %}
-{% set version = "2.7.0" %}
-{% set sha256 = "9c3bdc06fe411d24015e8bbbab9d03dc5243a970154496aac13f9283682435f9" %}
+{% set version = "2.3.0" %}
```

Comment:
Why downgrade?

Reply:
@jakirkham: I need two versions. First I built 2.7.0, and now I am building 2.3.0.